README.md

```diff
@@ -9,6 +9,8 @@ This role is needed to get access for Extra Packages for Enterprise Linux (EPEL)
 
 ## Installation
 
+**NOTE**: The armhfp repository is automatically rebuilt without any QA or tests. Please set ```wtd_repo_epel_armhfp_confirm``` to ```true```.
```

Comment:
regarding the false positives in the spellcheck: please use single backticks for inline codeblocks.
After merging the below pull request, they should be ignored correctly.
while-true-do/ansible-galaxy-skeleton#44

QA => quality assurance

"Please set variable to true if you want to use this repository."